templates/github/.github/workflows/release.yml.j2

```diff
@@ -115,8 +91,8 @@ jobs:
 
       {{ install_httpie() | indent(6) }}
 
-      {{ setup_env() | indent(6) }}
-
+      # Building the bindings and docs requires accessing the OpenAPI specs endpoint, so we need to
+      # setup the Pulp instance.
```

Comment:
Actually, the content-app container should be sufficient... :thinking:

Reply:
Do you mean the pulp-api app container?

Reply:
Oh, sorry, unrelated. I was just thinking what the minimal requirement were to get the api.json. A container allowing to run pulpcore-manager should be sufficient, but in the ci we always fire up the complete single container. So please ignore!